static/app/routes.tsx

```diff
@@ -60,6 +59,19 @@ const hook = (name: HookName) => HookStore.get(name).map(cb => cb());
 
 const SafeLazyLoad = errorHandler(LazyLoad);
 
+type PromisedImport<C> = Promise<{default: C}>;
+type ComponentType = React.ComponentType<any>;
+
+/**
+ * Factory function to produce a component that will render the SafeLazyLoad
+ * _with_ the required props.
+ */
+function make<C extends ComponentType>(resolve: () => PromisedImport<C>) {
```

Comment:
Can maybe rename this later to something better

Reply:
yes the name should be more descriptive

Reply:
I named it this since this file only really does one thing _make_ routes, lol

Reply:
Could call it `loadRoute()` or `importView()` :shrug:

Reply:
I just wanted it to be short so it doesn't wrap too many entires 😅

I figure since it's one of the _only_ functions in this file, a short name is OK -- since it's used like 300 times